client/human/HumanClientApp.cpp

```diff
@@ -1127,3 +1128,44 @@ HumanClientApp* HumanClientApp::GetApp()
 
 void HumanClientApp::Initialize()
 {}
+
+void HumanClientApp::OpenURL(const std::string& url) {
```

Comment:
OK, after some searching, I can't find such a library, which is both astonishing and disconcerting...  My second point still stands, though.  We should probably at least use a regular expression to make sure it's a valid URL.